duckduckgo/Android
# Smartling Translations

Task/Issue URL: 
- https://app.asana.com/0/414730916066338/1202390981544276/f
- https://app.asana.com/0/414730916066338/1202420653283990/f

### Description
Updated translations in all languages for App Tracking Protection and updated dax dialog copy

### Steps to test this PR

- Install from this branch
- Change device language to any (but English)
- Go to Overflow menu > Settings
- [ ]  Check App Tracking Protection title is translated
- [ ]  Check App Tracking Protection description is translated

### No UI changes
